app/helpers/database.js

```diff
@@ -6,7 +6,18 @@ var mongoose = require('mongoose'),
 	flash = require('connect-flash'),
 	session = require('express-session'),
 	log = require('./logger');
```

Comment:
Should change the `;` at the end of this line to a `,` so `MongoDBStore` is declared properly